src/BattleProtos.cs

```diff
@@ -122,49 +122,49 @@ public static void AddProtos()
             ProtoRegistry.RegisterString("即将到来gm", "Coming soon", "即将推出");
 
 
-            ProtoRegistry.RegisterString("彩蛋1描述", "Seems like these enemies are a kind of space insects. They live in deep space, feed on the tides of power without vision. " +
-                "The stellar station will create regular ripples in the space, making itself as a beacon in dark universe, can be easily captured and attacked by the insects. " +
-                "In other words, <color=#c2853d>they will randomly choose a stellar station, and attack its solar system by wormholes. Stars with more stellar stations will have higher possibility to be selected, and stars without any stellar stations won't be attacked. </color>" +
-                "Obviously, I can't rely on giving up stellar stations, as the progress will come slowly without transportations. Instead, I have to find a way to defense. Hope I still have time to think about 'The Answer to Life, the Universe, and Everything'.", 
+            ProtoRegistry.RegisterString("彩蛋1描述", "Seems like these enemies are a kind of space insect. They live in deep space, feed on the tides of power without vision. " +
+                "The interstellar station creates regular ripples in space, making itself a beacon in a dark universe that can easily be captured and attacked by the insects. " +
```

Comment:
Here I assume you're referring [to an ILS](https://dyson-sphere-program.fandom.com/wiki/Interstellar_Logistics_Station)?